test/t/test_gcc.py

```diff
@@ -1,7 +1,64 @@
 import pytest
 
+from conftest import assert_bash_exec
+
 
 class TestGcc:
+    @pytest.fixture(scope="class")
+    def gcc_with_completion(self, bash):
+        got = assert_bash_exec(bash,
+                               "gcc --help=common | grep completion || :",
+                               want_output=True)
```

Comment:
I think this will fail always when there's no `completion` in the output, because there's no completion at all then, but `want_output=True` insists there must be.

Reply:
You are right, I fixed that and it seems that it runs both of new and an older GCC release.